### DISCUSSION TITLE
Can this tool be used for youtube videos?

### DISCUSSION THREAD

**Discussion by darmoktan** [KevinBytesTheDust/CVAmp#225] [General]
I read that this supports youtube, but I saw no documentation about it.
I'm not interested in boosting a livestream but rather some youtube videos, using paid proxies of course.

If need be, I can purchase the ko-fi version as well.

**Comment by jlplenio** [comment]
It should work on YouTube live streams. I have not tested it for videos - and it is not really designed for that.
You could spawn and destroy instances to increase the "viewed count", but I assume there are better methods out there.
I do not plan to implement this.
